vdc/gcp-config.yaml.in

```diff
@@ -1,33 +1,39 @@
 resources:
 # service accounts
-- name: vdc-service-account
+- name: vdc-sa
   type: iam.v1.serviceAccount
   properties:
-    accountId: vdc-svc
-    displayName: for vdc-svc
+# must be 6-30 characters
+    accountId: vdc-sa
+    displayName: for vdc
 - name: gcr-push
   type: iam.v1.serviceAccount
   properties:
     accountId: gcr-push
     displayName: push to gcr.io
-- name: gcr-push-key
-  type: iam.v1.serviceAccounts.key
-  metadata:
-    dependsOn:
-    - gcr-push
+- name: gcr-pull
+  type: iam.v1.serviceAccount
+  properties:
+    accountId: gcr-pull
+    displayName: pull to gcr.io
+- name: k8s-admin
+  type: iam.v1.serviceAccount
   properties:
-    parent: projects/hail-vdc/serviceAccounts/gcr-push@hail-vdc.iam.gserviceaccount.com
+    accountId: k8s-admin
+    displayName: pull to gcr.io
+# buckets
+# deployment manager cannot create artifacts.@project@.appspot.com
 # sql instance
 - name: sql-instance
```

Comment:
What are the motivations for mysql vs Postgres? I can think of the uber engineering blog post from '16. Other than that, I don't have much background in either and no reason to prefer one to the other. If it's an arbitrary choice, that's fine too.

Reply:
I have no knowledge to prefer on over the other, arbitrary.  If you have any  reason to prefer Postgres, happy to switch now.